types/p5/src/color/setting.d.ts

```diff
@@ -249,6 +249,23 @@ declare module '../../index' {
          */
         clear(r: number, g: number, b: number, a: number): p5;
 
+        /**
+         *   Clears the pixels on the canvas. This function
+         *   makes every pixel 100% transparent. Calling
+         *   clear() doesn't clear objects created by createX()
+         *   functions such as createGraphics(), createVideo(),
+         *   and createImg(). These objects will remain
+         *   unchanged after calling clear() and can be
+         *   redrawn. In WebGL mode, this function can clear
```

Comment:
Maybe remove bit in comment about the parameters, given it's not relevant to this version?

Reply:
Thanks for your feedback, I agree it would make sense but it looks like the convention for p5 function is to keep the complete description and remove the `@params` lines as I have done here.

Some examples include [`pixelDensity`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3d0e1ffdb0da738be1777097e6484dbf799e7dff/types/p5/global.d.ts#L2546-L2566) and [`frameRate`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3d0e1ffdb0da738be1777097e6484dbf799e7dff/types/p5/global.d.ts#L2454-L2506).

So I'll keep it this way to maintain consistency.